test/pipelines-it-pai.yml

```diff
@@ -51,9 +51,9 @@ jobs:
 
       echo "TEST_IMG:$TEST_IMG"
       cd test
-      python3 generate_ts_config.py --ts pai --pai_host $(pai_host) --pai_user $(pai_user) --pai_pwd $(pai_pwd) --vc $(pai_virtual_cluster) \
-      --nni_docker_image $TEST_IMG --data_dir $(data_dir) --output_dir $(output_dir) --nni_manager_ip $(nni_manager_ip)
+      python3 generate_ts_config.py --ts pai --pai_host $(pai_host) --pai_user $(pai_user) --nni_docker_image $TEST_IMG --paiStoragePlugin $(paiStoragePlugin)\
+       --pai_token $(pai_token) --nniManagerNFSMountPath $(nniManagerNFSMountPath) --containerNFSMountPath $(containerNFSMountPath) --nni_manager_ip $(nni_manager_ip)
 
-      PATH=$HOME/.local/bin:$PATH python3 config_test.py --ts pai
+      PATH=$HOME/.local/bin:$PATH python3 config_test.py --ts pai --exclude multi_phase
```

Comment:
Why include multi phase?

Reply:
paiK8STrainingService does not support multi phase now, maybe will support in next release.